src/OpenRiaServices.Tools/Framework/OpenRiaServices.Tools.csproj

```diff
@@ -18,6 +34,14 @@
     <Reference Include="System.ServiceModel.Web" />
     <Reference Include="System.Web" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- Use PrivateAssets="all" to prevent nuget packages to become nuget dependencies -->
+    <!-- IMPORTANT: Also use ExcludeAssets="Runtime" if referencing msbuild packages https://learn.microsoft.com/en-us/visualstudio/msbuild/tutorial-custom-task-code-generation?view=vs-2022#dont-bundle-the-microsoftbuildutilitiescore-assembly -->
+    <PackageReference Include="Mono.Cecil" Version="0.11.5" PrivateAssets="all" />
```

Comment:
todo: look at excluding "Cecil.mdb" or if there is some way to include just "Mono.Cecil" and "Mono.Cecil.Pdb"